docs/demo/Aries-Workshop.md

```diff
@@ -80,10 +80,6 @@ That's it--you should be ready to start issuing and receiving verifiable credent
 
 ## Lab 2: Getting Ready To Be An Issuer
 
-::: todo
-To Do: Update lab to use this schema: H7W22uhD4ueQdGaGeiCgaM:2:student id:1.0.0
-:::
-
```

Comment:
Saw all schema references in this doc already using `H7W22uhD4ueQdGaGeiCgaM:2:student id:1.0.0` (and used that one while testing the instructions) so I don't _think_ this is needed?

Reply:
Sounds like it should be removed.